client/blocks/conversation-caterpillar/index.jsx

```diff
@@ -21,19 +22,22 @@ class ConversationCaterpillarComponent extends React.Component {
 
 	render() {
 		const { comments, translate } = this.props;
-		const lastComment = last( comments );
-		const lastCommenterName = get( lastComment, 'author.name' );
-		const commentCount = comments.length;
+		const commentCount = size( comments );
+
+		// Only display authors with a gravatar, and only display each author once
+		const uniqueAuthors = uniqBy( map( comments, 'author' ), 'ID' );
+		const displayedAuthors = filter( uniqueAuthors, author => !! author.avatar_URL );
```

Comment:
Ah - won't `reject` give us the elements for which `author.avatar_URL` is falsey? I suspect we do want `filter` after all.

I'll use the property shorthand though!